api/src/admins/admins.controller.ts

```diff
@@ -0,0 +1,122 @@
+import {
+  Controller,
+  Get,
+  NotFoundException,
+  Param,
+  ParseIntPipe,
+  Query,
+  HttpException,
+  Post,
+  Body,
+  ForbiddenException,
+  UseGuards,
+} from '@nestjs/common';
+import { Admin } from 'src/types';
+import { AdminsService } from './admins.service';
+import { ADMIN_TYPE } from 'src/constants';
+import { JwtAuthGuard } from '../auth/jwt-auth.guard';
+
+@UseGuards(JwtAuthGuard)
```

Comment:
I don't remember why you didn't add another Guard for admin. It would be cleaner, keep it in mind, maybe try to add it after the demo?